docs/en/compute/table.md

```diff
@@ -0,0 +1,224 @@
+databases from Alluxio.
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table
+Usage: alluxio table [generic options]
+         [attachdb <dbName> <dbType> [-o|--option <key=value>]]
+         [detachdb <db name>]
+         [ls [<database_name> [<table_name>]]]
+```
+
+To attach a database use the `attachdb` command. Currently, only `hive` is supported as the
+`dbType`.
+This command maps the hive database `default` into a database in Alluxio called `alluxio_db` from
+the metastore located at `thrift://metastore_host:9083`
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table attachdb alluxio_db hive \
+    --option udb-hive.database-name=default \
+    --option udb-hive.metastore.uris=thrift://metastore_host:9083
+```
+
+### Exploring Attached Databases
+
+Once attached, check that the database has been mounted with `alluxio table ls`
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table ls
+alluxio_db
+```
+
+List the tables underneath the database with `alluxio tables ls <db_name>`.
+If any tables exist underneath the corresponding database in hive, they will appear when executing
+this command.
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table ls alluxio_db
+test
+```
+
+In this case there is a table `test` in the `alluxio_db`.
+To get more information about a table in a database, run
+```console
+$ alluxio table ls <db_name> <table_name>
+```
+This command will dump information about the table to the console.
+An example of the output is below:
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table ls alluxio_db test
+db_name: "alluxio_db"
+table_name: "test"
+owner: "alluxio"
+schema {
+  cols {
+    name: "c1"
+    type: "int"
+    comment: ""
+  }
+}
+layout {
+  layoutType: "hive"
+  layoutSpec {
+    spec: "test"
+  }
+  layoutData: "\022\004test\032\004test\"\004test*\345\001\n\232\001\n2org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe\022(org.apache.hadoop.mapred.TextInputFormat\032:org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat\0227alluxio://localhost:19998/catalog/test/tables/test/hive\032\v\020\377\377\377\377\377\377\377\377\377\001 \0002\v\022\002c1\032\003int*\000"
+}
+parameters {
+  key: "totalSize"
+  value: "6"
+}
+parameters {
+  key: "numRows"
+  value: "3"
+}
+parameters {
+  key: "rawDataSize"
+  value: "3"
+}
+parameters {
+  key: "COLUMN_STATS_ACCURATE"
+  value: "{\"BASIC_STATS\":\"true\"}"
+}
+parameters {
+  key: "numFiles"
+  value: "3"
+}
+parameters {
+  key: "transient_lastDdlTime"
+  value: "1571191595"
+}
+```
+
+### Detaching databases
+
+If desired, databases can be detached from the Alluxio namespace by using
+
+```console
+$ alluxio table detach <database name>
+```
+
+For the previous examples, to detach we would run:
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table detachdb alluxio_db
+```
+
+Running `alluxio table ls` afterwards will not display the database any more.
+Continue to the next section to see how to use {{feature_name}} with presto
+
+## Data Transformations
```

Comment:
I'd recommend taking Data Transformation out of the Catalog Service docs. It should live independent of the catalog and include information about how the catalog knows which target data version to use.